public/locales/en/translation.json

```
@@ -1782,6 +1807,7 @@
     "actions": "Action Items",
     "search": "Search",
```

Biome (2.1.2)

[error] The key search was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)